package.json

```diff
@@ -31,6 +31,9 @@
     "istanbul": "^0.4.5",
     "istanbul-coveralls": "^1.0.3",
     "nyc": "^11.3.0",
+    "pg": "^7.4.0",
+    "pg-hstore": "^2.3.2",
+    "sequelize": "^4.23.1",
     "validator": "^9.1.1"
   },
   "engines": {
```

Comment:
why do u have this option `engines` ?

Reply:
that was a mistake when i was trying to make code climate work and I forgot to remove it. I will rectify